store/views.py

```diff
@@ -77,6 +132,52 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    book_id = request.POST.get('bid')
+    print(book_id)
+    book = BookCopy.objects.get(id=book_id)
+    user = User.objects.get(username = request.user.username)
+    if(book!=0):
+        book.borrower = None
+        book.status = True
+        book.borrow_date = None
+        book.save()
+        response_data = {
+        'message': 'success',
+        }
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.